modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -19,6 +19,19 @@ import {Editor} from './Editor';
 
 import '../css/main.scss';
 
+const TOOLBAR_PANEL_OTHER_CONFIGS = {
+	toolbarImage: {
+		widgets: 'image,image2',
+	},
+	toolbarTable: {
+		cssSelector: 'td',
+	},
+	toolbarText: {cssSelector: '*'},
```

Comment:
`cssSelector: '*'` is causing this bug as described in PR's description:


https://user-images.githubusercontent.com/7663513/128943998-1942cab7-2b0d-430f-a907-1f589418ff2d.mov

Reply:
I guess that you mean that we can not set an image to be italic.

It's a bug, the image toolbar should appear, right?

Reply:
> It's a bug, the image toolbar should appear, right?

Did you removed `toolbarImage` property value in `CKEditorBalloonEditorConfigContributor.java`?

Reply:
🏃 

I see, so the issue is that what should appear when there isn't a toolbarImage plugin?

Reply:
In order to solve the ticket, toolbarImage toolbar will not be shown when removing from CKEditorBalloonEditorConfigContributor.

This thread is about another (possible) issue that I notice. When disabling toolbarImage plugin, We can select an image and a toolbarText is being shown. Same can be observed when disabling toolbarTable and selecting table cells, it will render the toolbarText but it's correct and it's working applying text styles for all/each cell(s) :)

Reply:
Yep, that's what I meant. I think it's Ok, it's the default toolbar.

Reply:
Yeah, the text toolbar as default seems fine. For image, it even contains almost all useful options, link and alignment. Maybe down the line we can improve by defining a toolbar for any context and a separate toolbar for text, but this is more than good enough for now.